client/src/components/pages/LogIn/index.js

```diff
@@ -0,0 +1,81 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import singUpValidation from '../utils/schema';
+import Input from '../../common/Input';
+import Button from '../../common/Button';
+
+import('./index.css');
+
+class LogIn extends Component {
+  state = {
+    username: '',
+    password: '',
+    error: false,
+  };
+
+  onChangHandler = ({ target: { name, value } }) => {
+    this.setState({
+      [name]: value,
+    });
+  };
+
+  handleSubmit = e => {
+    e.preventDefault();
+    const { username, password } = this.state;
+    singUpValidation.isValid({ username, password }).then(res => {
+      if (!res) {
+        this.setState({
+          error: true,
+        });
+      } else {
+        this.setState({
+          error: false,
+        });
+        // axios stuff goes here
+      }
+    });
+  };
+
+  render() {
+    const { error, password, username } = this.state;
+    const errorMessage = error ? (
+      <p className="error-message">Please correct your email or password</p>
+    ) : null;
+    return (
+      <div className="login">
+        <p className="login__title">Welcome to parent assistent system</p>
+        <form className="login__form" onSubmit={this.handleSubmit}>
+          <p className="login__header">LOGIN</p>
+
+          <Input
+            name="username"
+            id="username"
+            value={username}
+            type="text"
+            placeholder="User name"
+            className="login__input"
+            onChange={this.onChangHandler}
```

Comment:
Consider this: `onChange={target => this.setState({ username: target.value })}`
Same thing for the password below. Then you can get rid of the `onChangeHandler` function and the `name` attributes and it becomes more readable - you see right away what is happening, instead of having to go through the onChangeHandler function and figuring out what the target's name is..